TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -289,11 +289,11 @@ void AddPresentItemLocations()
 			Add(new ItemKey(11, 25, 104, 192), "Below lab entrance", ItemProvider.Get(EItemType.MaxAura), TheLab & R.DoubleJump);
 			Add(new ItemKey(11, 18, 824, 128), "Trash jump room", ItemProvider.Get(EInventoryUseItemType.ChaosHeal), TheLabPoweredOff);
 			Add(new RoomItemKey(11, 39), "Dynamo Works", ItemProvider.Get(EInventoryOrbType.Eye, EOrbSlot.Melee), TheLabPoweredOff);
-			Add(new RoomItemKey(11, 21), "Blob mom", ItemProvider.Get(EInventoryRelicType.ScienceKeycardA), UpperLab);
+			Add(new RoomItemKey(11, 21), "Genza", ItemProvider.Get(EInventoryRelicType.ScienceKeycardA), UpperLab); //use their actual name for consistency
 			Add(new RoomItemKey(11, 1), "Experiment #13", ItemProvider.Get(EInventoryRelicType.Dash), TheLabPoweredOff);
-			Add(new ItemKey(11, 6, 328, 192), "Download and chest room", ItemProvider.Get(EInventoryEquipmentType.LabCoat), UpperLab);
+			Add(new ItemKey(11, 6, 328, 192), "Genza's closet", ItemProvider.Get(EInventoryEquipmentType.LabCoat), UpperLab); //"Chest and download room" could be anywhere
```

Comment:
This is the only place that has both a download and a chest in the same room except for the Library.